back/app/serializers/web_api/v1/initiative_serializer.rb

```diff
@@ -36,9 +36,18 @@ class WebApi::V1::InitiativeSerializer < WebApi::V1::BaseSerializer
     can_moderate?(object, params)
   }
 
+  attribute :cosponsorships do |object, params|
+    name_service = UserDisplayNameService.new(AppConfiguration.instance, current_user(params))
+
+    object&.cosponsors_initiatives&.map do |ci|
+      { user_id: ci.user_id, name: name_service.display_name!(ci.user), status: ci.status }
+    end
+  end
+
   has_many :initiative_images, serializer: WebApi::V1::ImageSerializer
   has_many :topics
   has_many :areas
+  has_many :cosponsors, record_type: :user, serializer: WebApi::V1::UserSerializer
```

Comment:
Isn't `cosponsorships` enough for all our cases? If enough, then we also don't need `include: %i[... cosponsors ...` in the controller.

`cosponsorships` and `cosponsors` currently duplicate one another.

Reply:
As explained in the PR description, `cosponsors` enables the inclusion of the related user records in the `includes` section of the response. `cosponsorships` is a custom serialized attribute that includes the user's display name && cosponsorship status.

Are you saying the FE will not need the full serialized user in the `includes` section of the response? (I'm guessing you might need the ID, in which case I could add it to the `consponsorships` attribute.

Reply:
Ahh, I think i get it. We can include users in the `cosponsorhips` attribute, as in your [other comment](https://github.com/CitizenLabDotCo/citizenlab/pull/5524#discussion_r1284653267).

Reply:
> Isn't cosponsorships enough for all our cases?

I forgot we may also need the user's avatar. So, indeed we need the users' objects here (as `cosponsors`).

If so, we do need `include: %i[... cosponsors ...` in the controller.

> cosponsorships and cosponsors currently duplicate one another.

Probably, it's fine and both will be used on the FE for the sake of simplicity

Reply:
OK, I'll put them back in 👍